src/GafferUI/DotNodeGadget.cpp

```diff
@@ -108,12 +125,18 @@ void DotNodeGadget::doRenderLayer( Layer layer, const Style *style, RenderReason
 		glPopMatrix();
 	}
 
-	NodeGadget::doRenderLayer( layer, style );
 }
 
 unsigned DotNodeGadget::layerMask() const
 {
-	return NodeGadget::layerMask() | (unsigned)GraphLayer::Nodes;
+	return NodeGadget::layerMask() | GraphLayer::Nodes | GraphLayer::OverBackdrops ;
+}
+
+Box3f DotNodeGadget::renderBound() const
+{
+	Box3f b = bound();
+	float maxThick = 2.0f;
+	return Box3f( b.min - V3f( maxThick, maxThick, 0.f ), b.max + V3f( maxThick, maxThick, 0.f ) );
```

Comment:
Oh, I guess it's true that the bound from StandardNodeGadget would work here ... it just felt like since I'm overriding doRenderLayer I should override the bound to match, but I guess there's no actual need for that in this case.